app/services/flow/flow_state_machine.rb

```diff
@@ -169,17 +169,13 @@ def flow_finish
       redirect_to send(@final_url)
     end
 
-    def redirect_to_step(step)
+    def redirect_to_step(_step)
       flow_finish and return unless next_step
-      redirect_url(step)
+      redirect_url
     end
 
-    def redirect_url(step)
-      if IdentityConfig.store.in_person_state_id_controller_enabled
-        redirect_to idv_in_person_proofing_state_id_url
-      else
-        redirect_to send(@step_url, step: step)
-      end
+    def redirect_url
+      redirect_to idv_in_person_proofing_state_id_url
```

Comment:
I have not dug into this yet but have concerns. If this is being used else where this would always direct to state_id. I will look into this more in a bit- just making a note of it so I don't forget to come back. (It might fine if nothing else is using this.)